src/consumer/component/link-generator.js

```diff
@@ -49,16 +48,20 @@ function _getLinkContent(filePath: string, importSpecifier?: Object): string {
         // @todo: implement
       }
     }
+    if (filePath.endsWith('.st.css')) {
+      // Wix Stylable format
+      return ':import { -st-from: "{filePath}.css"; }';
+    }
     return LINKS_CONTENT_TEMPLATES[fileExt];
   };
 
   if (!filePath.startsWith('.')) {
     filePath = `./${filePath}`; // it must be relative, otherwise, it'll search it in node_modules
   }
 
-  filePath = getWithoutExt(filePath); // remove the extension
+  const filePathWithoutExt = getWithoutExt(filePath);
```

Comment:
We should also update the logic in getWithoutExt to get the file without the .st.css (now it will only remove the .css)

Reply:
Agree. Will be changed.